core/plannernotes.c

```diff
@@ -30,16 +30,65 @@ int diveplan_duration(struct diveplan *diveplan)
 	return (duration + 30) / 60;
 }
 
+struct icd_data { // This structure provides communication between function isobaric_counterdiffusion() and the calling software.
+	int time; 			// For a trimix gas change, the time (in seconds) of the change
+	int ambientpressure_mbar;	// The ambient pressure (mbar) when the gas change took place
+	int dN2;			// The change in fraction (permille) of nitrogen during the change
+	int dHe;			// The change in fraction (permille) of helium during the change
+};
+
+/* Perform isobaric counterdiffusion calculations for gas changes in trimix dives.
+ * Here we use the rule-of-fifths where, during a change involving trimix gas, the increase in nitrogen
+ * should not exceed one fifth of the decrease in helium.
+ * Parameters:	1) Pointer to the dive structure.
+ *				2) Pointer to the dive planner point where the gas change is taking place.
+ *				3) pointers to two gas mixes, the gas being switched from and the gas being switched to.
+ * 				4) a pointer to an icd_data structure.
```

Comment:
shouldn't we align the above comment lines `2) ... 3)...etc` to start with the same indentation as `1)`.
this should be done with spaces instead of tabs.

Reply:
Please see what it looks like now? There is a discrepancy between the rendering of tabs between gedit that I use, and the way that the gitbub editor rends the same contents. See image below in gedit which shows good alignment. I can do some final edits in the github interactive editor. Would this be a way out?
![tab-rendering](https://user-images.githubusercontent.com/5078706/35140982-e4634bac-fd01-11e7-9183-3ebf6551f746.png)